Source/Core/DolphinWX/NetWindow.cpp

```diff
@@ -97,11 +97,47 @@ static void FillWithGameNames(wxListBox* game_lbox, const CGameListCtrl& game_li
 NetPlaySetupDiag::NetPlaySetupDiag(wxWindow* const parent, const CGameListCtrl* const game_list)
 	: wxFrame(parent, wxID_ANY, _("Dolphin NetPlay Setup"))
 	, m_game_list(game_list)
+	, m_central_server("")
+	, m_central_port(0)
 {
 	IniFile inifile;
 	inifile.Load(File::GetUserPath(D_CONFIG_IDX) + "Dolphin.ini");
 	IniFile::Section& netplay_section = *inifile.GetOrCreateSection("NetPlay");
 
+	// Check traversal ini
```

Comment:
If your talking about the check for empty strings, I mainly did that because the previous version set the traversal server and port to empty before and I would like to fix that for anyone who has the previous version.  If that's not what you are talking about, is there a proper or better way I could emulate?